src/utils/dataset-controller-utils.ts

```diff
@@ -31,7 +31,7 @@ export const addDirectoryToZip = async (
 
 export const collectFiles = (dataset: Dataset): Map<string, FileImportDto> => {
   const files = new Map<string, FileImportDto>();
-  if (dataset.measure.lookupTable) {
+  if (dataset.measure && dataset.measure.lookupTable) {
```

Comment:
```suggestion
  if (dataset.measure?.lookupTable) {
```
... or use optional chaining.